versioned_docs/version-1.4/managedosversionchannel-reference.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_label: ManagedOSVersionChannel reference
+title: ''
+---
+
+<head>
+  <link rel="canonical" href="https://elemental.docs.rancher.com/managedosversionchannel-reference"/>
+</head>
+
+# ManagedOSVersionChannel reference
+
+The `ManagedOSVersionChannel` resource is responsible of defining OS image channel.
+
+There are several keys that can be configured under a `ManagedOSVersionChannel` resource spec.
+
+```yaml title="managedosversionchannel-example.yaml" showLineNumbers
+apiVersion: elemental.cattle.io/v1beta1
+kind: ManagedOSVersionChannel
+metadata:
+  name: elemental-channel
+  namespace: fleet-default
+spec:
+  options:
+    image: registry.suse.com/rancher/elemental-channel:1.4.2
+  syncInterval: 1h
+  type: custom
+```
+
+#### ManagedOSVersionChannelSpec reference
+
+| Key              | Type   | Default value | Description                                                                  |
+|------------------|--------|---------------|------------------------------------------------------------------------------|
+| options          | object | null          | Defines the optional informations that can be added in an OS channel         |
+| syncInterval     | string | empty         | Defines when to update the OS channel                                        |
+| type             | string | empty         | Defines the channel type, only `custom` is supported now                     |
+| upgradeContainer | object | null          | An upgrade container that can be defined. See [reference](#upgradecontainer) |
+
+#### upgradeContainer
+
+This allows to overwrite the default `upgrade` container executing the upgrade job on a node.
+The keys directly translate to the [container](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#Container) specification.  
```

Comment:
I don't think this is accurate enough, in fact `upgradeContainer` is of type `ContainerSpec` defined in system-upgrade-controller. Unless you verified system-upgrade-controller translates directly to the k8s pod resource (which is provably the case) I'd just mention this of type `ContainerSpec`, part of System Upgrade Controller project, and maps the the `ugprade` field of a System Upgrade Controller Plan. What about something in the spirit of:

`upgradeController` is mapped to a `v1.ContainerSpec` object (defined as part of [System Upgrade Controller](https://github.com/rancher/system-upgrade-controller)), which will be used as the `upgrade` field in an eventual System Upgrade Controller Plan created by the elemental-operator when upgrading to a `ManagedOSVersion` defined from this channel.